src/styles/main.scss

```diff
@@ -1,3 +1,44 @@
-body {
-  margin: 0;
+@import "./reset.scss";
+@import "./variables.scss";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $calendar-padding;
+  max-width: $day-width * 7 + $calendar-padding * 2 + $gap * 6;
+
+  &__day {
+    display: flex;
+    background-color: #eee;
+    flex-basis: $day-width;
+    height: $day-height;
```

Comment:
why not just width and height

Reply:
because, in the future, the day could become a rectangle.